engine/lib/navigation.php

```diff
@@ -510,6 +510,10 @@ function _elgg_river_menu_setup($hook, $type, $return, $params) {
 					'title' => elgg_echo('comment:this'),
 					'rel' => 'toggle',
 					'priority' => 50,
+					'data-animation' => json_encode([
+						'open' => 'slideInDown',
+						'close' => 'slideOutUp',
+					]),
```

Comment:
Animate.css looks a poor replacement for $ slideDown/Up, i'm guessing because it can't animate height. Throughout the animation the full height is present, creating abrupt transitions at the beginning of an "in" and the end of an "out". Not necessarily a deal-breaker.